### ISSUE TITLE
API Base URL cofiguration

### ISSUE THREAD

**Issue by willbreitkreutz** [USACE/groundwork#51]
Not something I'd halt merge for, but do we need a consistent way to expose API Roots so that they can be used externally / overridden by a consuming app?

_Originally posted by @willbreitkreutz in https://github.com/USACE/groundwork/pull/49#discussion_r1589559260_

Adding this for further conversation.  I'm going to merge the existing changes now and we can revisit with a more holistic approach.

**Comment by jbkolze** [comment]
Related to #26 
I thought about this as I was building the NWPS hooks.  I think it makes sense for APIs that we control (e.g. CDA) for development purposes, but I think it would be rare to have a situation in which we would be using different URLs for the same external API.  Occasionally the URL will need to be updated, such as when NWPS went from dev to prod, but that's a change that would theoretically be required across-the-board for anyone using groundwork.

**Comment by willbreitkreutz** [comment]
Yeah, my only thought was a single place for them all, if they update regularly or not...
